Assets/Scripts/Ghost/ThirdPersonController.cs

```diff
@@ -15,11 +17,9 @@ namespace StarterAssets
     public class ThirdPersonController : MonoBehaviour
     {
         [Header("Player")]
-        [Tooltip("Move speed of the character in m/s")]
         public float MoveSpeed = 2.0f;
```

Comment:
Too much public variables. Check each variable if you really need them to be public